scico/linop/xray/_xray.py

```diff
@@ -20,22 +19,25 @@
 
 from scico.typing import Shape
 
-from ._linop import LinearOperator
+from .._linop import LinearOperator
 
 
-class XRayProject(LinearOperator):
-    """X-ray projection operator.
+class XRayTransform(LinearOperator):
+    """X-ray transform operator.
 
-    Wraps an X-ray projector object in a SCICO
-    :class:`LinearOperator`.
+    Wrap an X-ray projector object in a SCICO :class:`LinearOperator`.
+    **Warning:**  Note that the only X-ray projector object currently
+    supported, :class:`.Parallel2dProjector`, is not a very accurate
+    approximation of the integral transform representing real projection
+    imaging, and is therefore not suitable for real imaging applications.
```

Comment:
"may not be suitable"? not trying to defend its accuracy, but we do we know it isn't suitable?